Piezo_Peak_Capture.ino

```diff
@@ -40,5 +40,15 @@ void loop() {
       state = 0; // go back to idle after 30 ms below threshold
     }
   }
+  float volume = 0.0;
```

Comment:
`float` is a type of variable that has a decimal. Things like `int` and `long` can't have any decimal part, so we have to use `float`